src/components/SpruceForm/Widgets/types.ts

```diff
@@ -16,6 +16,7 @@ export interface SpruceWidgetProps extends WidgetProps {
     inputType: TextInputType;
     warnings: string[];
     customLabel: string;
+    sizeVariant: string;
```

Comment:
I noticed that too but I thought since this interface is generic to all widgets, it may not be appropriate to type it as a segmented control-specific `Size` (LeafyGreen uses different variants for all size props). 🤷‍♀️

Reply:
each of the LG components maintain their own unique "Size"s so I think it might not be possible. for example `SegmentedControl` does not have "small" size, but `Button` does

Reply:
good catch that makes sense to me!